odoo/addons/base/wizard/base_export_language.py

```diff
@@ -41,6 +42,8 @@ def act_getfile(self):
 
         with contextlib.closing(io.BytesIO()) as buf:
             if this.export_type == 'model':
+                if not this.model_name:
+                    raise UserError(_("Model is required to export the translation"))
```

Comment:
Isn't it better to make the field model_id required in the view when the `export_type == 'model'` ? :thinking: 
Also it avoids some work to the translation team.